app/models/miq_action.rb

```diff
@@ -20,7 +20,8 @@ class MiqAction < ApplicationRecord
               "set_custom_attribute"    => "Set a Custom Attribute in vCenter",
               "inherit_parent_tags"     => "Inherit Parent Tags",
               "remove_tags"             => "Remove Tags",
-              "delete_snapshots_by_age" => "Delete Snapshots by Age"
+              "delete_snapshots_by_age" => "Delete Snapshots by Age",
```

Comment:
@lgalis this change doesn't look related to addition of key to permissions file.